task3.py

```diff
@@ -0,0 +1,13 @@
+"""Напишите функцию, которая принимает строку и возвращает кол-во заглавных и строчных букв.
+'The quick Brow Fox' => Upper case characters: 3, Lower case characters: 12
+"""
+
+
+def count_upper_lower(simple_str):
+    upper_count = sum(1 for char in simple_str if char.isupper())
+    lower_count = sum(1 for char in simple_str if char.islower())
+    return f"Upper case characters: {upper_count}, Lower case characters: {lower_count}"
```

Comment:
Better to use 1 loop, instead of 2
```suggestion
    for char in string:
        if char.isupper():
            upper_count += 1
        elif char.islower():
            lower_count += 1
```